typescript/src/test_util.ts

```diff
@@ -5,11 +5,14 @@ export function setup() {
 }
 
 export function mockFetch(ok: boolean, data: any, status: number) {
-  (global as any).fetch = jest.fn().mockImplementation(() =>
+  const mock = jest.fn();
```

Comment:
I personally favors the old way you do it, but I do not have strong personal opinion about it